cwms-data-api/src/main/java/cwms/cda/data/dao/location/kind/LockDao.java

```diff
@@ -238,84 +193,76 @@ static Lock map(LOCK_OBJ_T lock, double upperWarningLevel, double lowerWarningLe
-                .withLockWidth(result.get(view.LOCK_WIDTH, Double.class))
-                .withNormalLockLift(result.get(view.NORMAL_LOCK_LIFT, Double.class))
-                .withVolumePerLockage(result.get(view.VOLUME_PER_LOCKAGE, Double.class))
-                .withMinimumDraft(result.get(view.MINIMUM_DRAFT, Double.class))
-                .withLengthUnits(result.get(view.LENGTH_UNIT_ID))
-                .withVolumeUnits(result.get(view.VOLUME_UNIT_ID))
-                .withHighWaterLowerPoolWarningLevel(result.get(view.ELEV_CLOSURE_HIGH_WATER_LOWER_POOL_WARNING,
-                        Double.class))
-                .withHighWaterUpperPoolWarningLevel(result.get(view.ELEV_CLOSURE_HIGH_WATER_UPPER_POOL_WARNING,
-                        Double.class))
-                .withChamberType(chamberType)
-                .withHighWaterLowerPoolWarningLevel(highWaterLowerPoolWarning)
-                .withHighWaterUpperPoolWarningLevel(highWaterUpperPoolWarning)
-                .withElevationUnits(result.get(view.ELEV_UNIT_ID))
-                .withVolumeUnits(result.get(view.VOLUME_UNIT_ID))
-                .withMaximumLockLift(result.get(view.MAXIMUM_LOCK_LIFT, Double.class))
-                .withLowWaterUpperPoolLocationLevel(lowWaterUpperPool != null
-                        ? new LockLocationLevelRef(mapToLockRef(lockLocation.getOfficeId(),
-                            String.format("%s.Elev-Closure.Inst.0.Low Water Upper Pool",
-                                    lockLocation.getName())), lowWaterUpperPool)
-                        : null)
-                .withLowWaterLowerPoolLocationLevel(lowWaterLowerPool != null
-                        ? new LockLocationLevelRef(mapToLockRef(lockLocation.getOfficeId(),
-                            String.format("%s.Elev-Closure.Inst.0.Low Water Lower Pool",
-                                    lockLocation.getName())), lowWaterLowerPool)
-                        : null)
-                .withHighWaterLowerPoolLocationLevel(highWaterLowerPool != null
-                        ? new LockLocationLevelRef(mapToLockRef(lockLocation.getOfficeId(),
-                            String.format("%s.Elev-Closure.Inst.0.High Water Lower Pool",
-                                    lockLocation.getName())), highWaterLowerPool)
-                        : null)
-                .withHighWaterUpperPoolLocationLevel(highWaterUpperPool != null
-                        ? new LockLocationLevelRef(mapToLockRef(lockLocation.getOfficeId(),
-                            String.format("%s.Elev-Closure.Inst.0.High Water Upper Pool",
-                                    lockLocation.getName())), highWaterUpperPool)
-                        : null)
-                .build();
-    }
-
     static String mapToLockRef(String office, String locationName) {
         return String.format("/locks/%s?office=%s", locationName, office);
     }
+
+    public Lock unitConvertToEN(Lock lock) {
```

Comment:
Add unit test to LockDaoTest for this method.

Reply:
Added test to LockDaoIT since DB calls are needed for this test